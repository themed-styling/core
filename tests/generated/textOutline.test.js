import 'jest-styled-components'

import { textOutline } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(textOutline, 'textOutline')
