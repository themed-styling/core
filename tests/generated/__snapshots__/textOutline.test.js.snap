// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textOutline renders 0 1`] = `
Array [
  .c0 {
  text-outline: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders function 1`] = `
Array [
  .c0 {
  text-outline: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: has;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: has;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders invalid theme path 1`] = `
Array [
  .c0 {
  text-outline: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders negative number 1`] = `
Array [
  .c0 {
  text-outline: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  text-outline: July;
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  text-outline: July;
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  text-outline: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-outline: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-outline: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    text-outline: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    text-outline: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-outline: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-outline: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-outline: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-outline: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  text-outline: July;
}

@media May {
  .c0 {
    text-outline: August;
  }
}

@media April {
  .c0 {
    text-outline: September;
  }
}

@media January {
  .c0 {
    text-outline: November;
  }
}

@media April {
  .c0 {
    text-outline: August;
  }
}

@media June {
  .c0 {
    text-outline: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  text-outline: July;
}

@media May {
  .c0 {
    text-outline: August;
  }
}

@media April {
  .c0 {
    text-outline: September;
  }
}

@media January {
  .c0 {
    text-outline: November;
  }
}

@media April {
  .c0 {
    text-outline: August;
  }
}

@media June {
  .c0 {
    text-outline: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  text-outline: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-outline: November;
  }
}

@media April {
  .c0 {
    text-outline: January;
  }
}

@media January {
  .c0 {
    text-outline: December;
  }
}

@media April {
  .c0 {
    text-outline: April;
  }
}

@media June {
  .c0 {
    text-outline: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-outline: November;
  }
}

@media April {
  .c0 {
    text-outline: January;
  }
}

@media January {
  .c0 {
    text-outline: December;
  }
}

@media April {
  .c0 {
    text-outline: April;
  }
}

@media June {
  .c0 {
    text-outline: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    text-outline: July;
  }
}

@media April {
  .c0 {
    text-outline: August;
  }
}

@media January {
  .c0 {
    text-outline: September;
  }
}

@media April {
  .c0 {
    text-outline: November;
  }
}

@media June {
  .c0 {
    text-outline: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    text-outline: July;
  }
}

@media April {
  .c0 {
    text-outline: August;
  }
}

@media January {
  .c0 {
    text-outline: September;
  }
}

@media April {
  .c0 {
    text-outline: November;
  }
}

@media June {
  .c0 {
    text-outline: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-outline: January;
  }
}

@media April {
  .c0 {
    text-outline: December;
  }
}

@media January {
  .c0 {
    text-outline: April;
  }
}

@media April {
  .c0 {
    text-outline: May;
  }
}

@media June {
  .c0 {
    text-outline: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-outline: January;
  }
}

@media April {
  .c0 {
    text-outline: December;
  }
}

@media January {
  .c0 {
    text-outline: April;
  }
}

@media April {
  .c0 {
    text-outline: May;
  }
}

@media June {
  .c0 {
    text-outline: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders postitive number 1`] = `
Array [
  .c0 {
  text-outline: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders string 1`] = `
Array [
  .c0 {
  text-outline: July;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: July;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: June;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: June;
}

<div
    className="c0"
  />,
]
`;

exports[`textOutline renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`textOutline renders valid theme path 1`] = `
Array [
  .c0 {
  text-outline: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-outline: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
